sandbox/settings.py

```diff
@@ -498,6 +498,12 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
     PAGINATION_DEFAULT_WINDOW = 2
     PAGINATION_DEFAULT_MARGIN = 1
 
+
+    # Django - core
+    # A boolean that specifies whether to display numbers using a thousand separator.
+    # Visible at least on the enrollment count of the course pages
+    USE_THOUSAND_SEPARATOR = True
```

Comment:
This is normally already taken care of by Django: https://github.com/django/django/blob/main/django/conf/locale/pt/formats.py#L34

Reply:
Yes, you are right. The solution was to use the [intcomma](https://docs.djangoproject.com/en/3.2/ref/contrib/humanize/#intcomma), to only format that specific number. But it requires to add `django.contrib.humanize` to the `INSTALLED_APPS`.